packages/fields-auto-increment/README.md

```diff
@@ -6,7 +6,7 @@ title: AutoIncrement
 
 # AutoIncrement
 
-An automatically incrementing integer with support for the Knex adapter. It's important to note that this type:
+An automatically incrementing integer with support for the Prisma adapter. It's important to note that this type:
```

Comment:
Lets not at this point, but eventually, we'll do a sweep and remove all references to "adapters" entirely.